src/LPOracleFactory.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.25;
+
+import { LPOracle } from "./LPOracle.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { ILPOracleFactory } from "./interfaces/IOracleFactory.sol";
+
+/// @title LPOracle Factory
+/// @notice Factory contract for deploying LPOracle instances with deterministic addresses
+contract LPOracleFactory is Ownable, ILPOracleFactory {
+    error OracleAlreadyExists();
+    error DeployFailed();
+
+    /// @notice BCoWHelper contract address used for all oracle deployments
+
```

Comment:
non-blocking: remove this line so natspec sits above storage variable.

Reply:
Oh I had a local commit with this fixed...